appengine-java11/pom.xml

```diff
@@ -0,0 +1,49 @@
+<!--
```

Comment:
This pom can be removed.

Reply:
Don't we need this for the tests to run properly ie parent v child pom?

Reply:
No - the update you made to Kokoro finds projects by using the pom's themselves instead of relying on the parent POM. 

(If we were still doing it with the parent pom this tests wouldn't trigger anyway because it doesn't list the sub projects as modules and isn't listed in the pom located at root)